grid/src/Cabana_Grid_BovWriter.hpp

```diff
@@ -393,18 +394,40 @@ void writeTimeStep( ExecutionSpace, const int time_step_index,
   This version writes a single output and does not use bricklets. We will do
   this in the future to improve parallel visualization.
 
+  \param prefix The filename prefix
   \param time_step_index The index of the time step we are writing.
   \param time The current time
   \param array The array to write
   \param gather_array Gather the array before writing to make parallel
   consistent.
 */
 template <class Array_t>
-void writeTimeStep( const int time_step_index, const double time,
-                    const Array_t& array, const bool gather_array = true )
+void writeTimeStep( const std::string& prefix, const int time_step_index,
+                    const double time, const Array_t& array,
+                    const bool gather_array = true )
+{
+    using exec_space = typename Array_t::execution_space;
+    writeTimeStep( exec_space{}, prefix, time_step_index, time, array,
+                   gather_array );
+}
+
+template <class ExecutionSpace, class Array_t>
+[[deprecated( "writeTimeStep now requires a filename prefix" )]] void
```

Comment:
Looking at this again, I would rather not deprecate and just directly use the array label as the prefix in the variant